### ISSUE TITLE
Good pretrained checkpoint model

### ISSUE THREAD

**Issue by MozhdehDokhani** [karpathy/neuraltalk2#202]
Hi,
Does anybody have a good pretrained checkpoint model here? I trained on MS COCO dataset for about 44 epochs, but the accuracy isn't good enough when I use it on some test images.
